support/routes/v2/isobot/index.ts

```diff
@@ -41,5 +42,26 @@ const handleWhitelistEmails: RequestHandler<
   }
 }
 
+const handleDnsChecker: RequestHandler<
```

Comment:
trouble you to remove the console.log in line 22 please

Reply:
Removed in 15d5512dc78e9bce861968a01f86caf51172741b!